src/contentMainScript.ts

```diff
@@ -104,6 +105,17 @@ function onConfigUpdate(e: CustomEvent<{ [CONFIG_MAX_PROMPT_WORDS]: number }>) {
   maxPromptWords = e.detail[CONFIG_MAX_PROMPT_WORDS] || maxPromptWords;
 }
 
+function onEditorAppend(
```

Comment:
nit: let's use "insert", since it's not always adding content to the end of the document.